src/models/Lessons.js

```diff
@@ -111,70 +111,95 @@ export async function generateLessons(sequelize, models) {
   return Lessons;
 }
 
-export function getAll() {
-  return Lessons.findAll()
-    .then(lessons => lessons)
-    .catch(() => 'fail');
+export async function getAll() {
+  try {
+    return await Lessons.findAll();
+  } catch (err) {
+    throw new Error(err);
+  }
 }
 
-export function getAllForCurrentSeminar(seminarId) {
-  return Lessons.findAll({ where: { seminar_id: seminarId } })
-    .then(lessons => lessons)
-    .catch(() => 'fail');
+export async function getAllForCurrentSeminar(seminarId) {
+  try {
+    return await Lessons.findAll({ where: { seminar_id: seminarId } });
+  } catch (err) {
+    throw new Error(err);
+  }
 }
 
-export function getByMonth(number) {
-  return Lessons.sequalize.query('SELECT * FROM lessons WHERE EXTRACT(MONTH FROM date) = :number',
-    { replacements: { number } },
-    { type: Lessons.sequalize.QueryTypes.SELECT })
-    .then(lessons => lessons[0])
-    .catch(() => 'fail');
+export async function getByMonth(number) {
+  try {
+    const lessons = await Lessons.sequalize.query('SELECT * FROM lessons WHERE EXTRACT(MONTH FROM date) = :number',
```

Comment:
Can we do same things without using `.query()` method of sequelize? Use methods findAll/findOne, please